recipe/meta.yaml

```diff
@@ -30,7 +31,6 @@ test:
   imports:
     - wheel
     - wheel.signatures
-    - wheel.test
```

Comment:
I think this should still be there. 😕

Reply:
I had removed that, but added that again. I still think wheel.test should not be there. In older version, there was a wheel.test directory, but not in the latest version.

Reply:
NVM. Appears I'm wrong. Seemed to be in the source and the `MANIFEST.in`, but not the tarball on PyPI. Sorry for the confusion. Please drop after all.